content/sigs/docs/gsod/2020/projects/document-jenkins-on-kubernetes.adoc

```diff
@@ -0,0 +1,80 @@
+---
+layout: gsodproject
+title: "Document Jenkins on Kubernetes"
+goal: "Create a new Kubernetes Volume which would describe the concepts, techniques, and choices for Kubernetes users running Jenkins."
+category: Documentation
+year: 2020
+sig: jenkins-documentation
+status: "Active"
+technicalWriter: zaycodes
+mentors:
+- "oleg_nenashev"
+- "MarkEWaite"
+- "kwhetstone"
+- "markyjackson-taulia"
```

Comment:
Author name is apparently case sensitive, so "MarkEWaite" needs to be "markewaite".
```suggestion
- oleg_nenashev
- markewaite
- kwhetstone
- markyjackson-taulia
```